kedro/contrib/io/parquet/dask_parquet_s3.py

```diff
@@ -0,0 +1,158 @@
+            >>>
+            >>> assert ddf.compute().equals(reloaded.compute())
+    """
+
+    # pylint: disable=too-many-arguments
+    def __init__(
+        self,
+        filepath: str,
+        bucket_name: str,
+        credentials: Optional[Dict[str, Any]] = None,
+        load_args: Optional[Dict[str, Any]] = None,
+        save_args: Optional[Dict[str, Any]] = None,
+        version: Version = None,
+    ) -> None:
+        """Creates a new instance of ``ParquetS3DaskDataSet`` pointing to a concrete
+        parquet file on S3.
+
+        Args:
+            filepath: Path to a parquet file
+                parquet collection or the directory of a multipart parquet.
+            bucket_name: S3 bucket name.
+            credentials: Credentials to access the S3 bucket, such as
+                ``aws_access_key_id``, ``aws_secret_access_key``.
+            load_args: Additional loading options `pyarrow`:
+                https://arrow.apache.org/docs/python/generated/pyarrow.parquet.read_table.html
+                or `fastparquet`:
+                https://fastparquet.readthedocs.io/en/latest/api.html#fastparquet.ParquetFile.to_pandas
+            save_args: Additional saving options for `pyarrow`:
+                https://arrow.apache.org/docs/python/generated/pyarrow.Table.html#pyarrow.Table.from_pandas
+                or `fastparquet`:
+                https://fastparquet.readthedocs.io/en/latest/api.html#fastparquet.write
+            version: If specified, should be an instance of
+                ``kedro.io.core.Version``. If its ``load`` attribute is
+                None, the latest version will be loaded. If its ``save``
+                attribute is None, save version will be autogenerated.
+        """
+
+        _credentials = deepcopy(credentials) or {}
+        _s3 = S3FileSystem(client_kwargs=_credentials)
+        super().__init__(
+            PurePosixPath("{}/{}".format(bucket_name, filepath)),
+            version,
+            exists_function=_s3.exists,
+            glob_function=_s3.glob,
+        )
+
+        default_load_args = {}  # type: Dict[str, Any]
+        default_save_args = {}  # type: Dict[str, Any]
+
+        self._load_args = (
+            {**default_load_args, **load_args}
+            if load_args is not None
+            else default_load_args
+        )
+        self._save_args = (
+            {**default_save_args, **save_args}
+            if save_args is not None
+            else default_save_args
+        )
+
+        self._bucket_name = bucket_name
+        self._credentials = _credentials
+        self._s3 = _s3
+
+    def _describe(self) -> Dict[str, Any]:
+        return dict(
+            filepath=self._filepath,
+            bucket_name=self._bucket_name,
+            load_args=self._load_args,
+            save_args=self._save_args,
+            version=self._version,
+        )
+
+    def _load(self) -> dd.DataFrame:
+        s3_file = f"s3://{self._bucket_name}/{self._filepath}"
```

Comment:
Since this class is inheriting from `AbstractVersionedDataSet`, you need to get the path from `self._get_load_path()`.  See how `ParquetS3DataSet` implements `_load`, and 

```python
load_path = PurePosixPath(self._get_load_path())

with self._s3.open(str(load_path), mode="rb") as s3_file:
    return pd.read_parquet(s3_file, **self._load_args)

```